### DISCUSSION TITLE
Blocks copy/paste and undo

### DISCUSSION THREAD

**Discussion by alexei-moneyr** [postmanlabs/postman-flows#162] [Ideas]
Pleeease add copy paste ability for blocks. 
In same flow and to a different flows.
Also undo is crucial. If I delete something, I have to remake it again.

Thanks

**Comment by saswatds** [comment]
@alexei-moneyr We have implemented this, it should be available in the next postman release.

**Reply by alexei-moneyr** [reply]
still not implemented ((( 
Since you changing ver a lot (not small updates) have to redo flows from scratch. And as for new flows, instead of copy past same BIG blocks, need to make them again (
Please add this... please...

Thanks

**Reply by saswatds** [reply]
Hey, @alexei-moneyr we implemented undo/redo. We are working on copy-paste as we speak
